docs/data_onboarding/network_devices/cloudflare.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Cloudflare
+permalink: /data_onboarding/network_devices/cloudflare/
+nav_order: 7
+parent: Network Devices
+grand_parent: Data Onboarding
+---
+
+## **Cloudflare Data**
+
+Cloudflare logs are collected via HEC (HTTP Event Collector) input. Refer the guide to onboard the logs: [https://developers.cloudflare.com/logs/get-started/enable-destinations/splunk/](https://developers.cloudflare.com/logs/get-started/enable-destinations/splunk/)
+
+Splunkbase Download (For Field Extractions): 
```

Comment:
This Add-on needs to be installed on SH or HF or both?

Reply:
Both